GVRf/Framework/framework/src/main/jni/backends/oculus/activity.cpp

```diff
@@ -140,10 +140,24 @@ void GVRActivity::onSurfaceChanged(JNIEnv& env) {
         if (mMultisamplesConfiguration > maxSamples)
             mMultisamplesConfiguration = maxSamples;
 
-        for (int eye = 0; eye < VRAPI_FRAME_LAYER_EYE_MAX; eye++) {
+
+        bool multiview;
+        configurationHelper_.getMultiviewConfiguration(env,multiview);
+
+        const char* extensions = (const char*)glGetString(GL_EXTENSIONS);
+        if(multiview && std::strstr(extensions, "GL_OVR_multiview2")!= NULL){
+            use_multiview = true;
+        }
+        if(multiview && !use_multiview){
+            std::string error = "Multiview is not supported by your device";
+            LOGE(" Multiview is not supported by your device");
+            throw error;
+        }
+
```

Comment:
This multiview flag is in too many places. We agreed that using a static is unavoidable at the moment. But it could be that there is a static in the Renderer class; a static that is set from here and this should be it. Or not?